exaudfclient/base/exaudflib/BUILD

```diff
@@ -90,7 +91,18 @@ cc_library(
 # depend directly on them
 cc_library(
     name = "exaudflib",
-    srcs = ["exaudflib.cc"],
+    srcs = ["exaudflib.cc",
+            "impl/msg_conversion.cc", "impl/msg_conversion.h",
```

Comment:
what was actually the solution to get this working

Reply:
Google recommends to include header files based on the workspace root directory: So I tried
```#include <exaudflib/impl/msg_conversion.h>``` which did not work.
Later I found (in https://docs.bazel.build/versions/main/bazel-and-cpp.html#include-paths) that it must be quotes (which actually I find odd because the clean way is to use < >):
So I changed all includes to
```#include "exaudflib/impl/msg_conversion.h"```

Reply:
This is because of the sandboxing